sql/14-increase_owner_pubkey.sql

```diff
@@ -8,5 +8,5 @@
 -- in a transaction
 BEGIN;
     ALTER TABLE allocations
-        ALTER COLUMN owner_public_key varchar(512) NOT NULL;
+        ALTER COLUMN owner_public_key TYPE varchar(512) NOT NULL;
```

Comment:
```
postgres-post_1  | /blobber/bin/postgres-entrypoint.sh: running /blobber/sql/14-increase_owner_pubkey.sql
postgres-post_1  | You are now connected to database "blobber_meta" as user "postgres".
postgres-post_1  | BEGIN
postgres-post_1  | psql:/blobber/sql/14-increase_owner_pubkey.sql:11: ERROR:  syntax error at or near "NOT"
postgres-post_1  | LINE 2: ...   ALTER COLUMN owner_public_key TYPE varchar(512) NOT NULL;
postgres-post_1  |                                                               ^
postgres_1       | 2021-06-01 15:25:01.695 UTC [107] ERROR:  syntax error at or near "NOT" at character 81
postgres_1       | 2021-06-01 15:25:01.695 UTC [107] STATEMENT:  ALTER TABLE allocations
postgres_1       |              ALTER COLUMN owner_public_key TYPE varchar(512) NOT NULL;
postgres-post_1  | ROLLBACK
postgres-post_1  | 
postgres-post_1  | 
blobber1_postgres-post_1 exited with code 0
```

Reply:
looks like NOT NULL is not needed, since that contraint is not changed by Postgres.